kiosk/src/Kiosk.css

```diff
@@ -50,7 +50,7 @@ h2 {
     text-align: center;
 }
 
-.sim-embed {
+#sim-embed {
```

Comment:
Is this classname to id change necessary or related to the focusing issue?

Reply:
Oh, I changed it because I was playing with a different way to  focus the iframe. I'll change it back, thanks!